setup.py

```diff
@@ -13,7 +13,7 @@
     description='Airbrake exception logger for Django',
     license='MIT',
     long_description=open('README.rst').read(),
-    install_requires=['Django>=1.8,<1.9.99'],
+    install_requires=['Django>=1.11,<=2.0.5'],
```

Comment:
Requiring up to 2.0.5 will cause problems once 2.0.6 or 2.1 is released. Probably better to not specify an upper bound.